reference/6/Microsoft.PowerShell.Core/About/about_CommonParameters.md

```diff
@@ -8,29 +8,29 @@ title:  about_CommonParameters
 # About CommonParameters
 
 ## SHORT DESCRIPTION
+
 Describes the parameters that can be used with any cmdlet.
 
 ## LONG DESCRIPTION
 
 The common parameters are a set of cmdlet parameters that you can use with any
-cmdlet. They are implemented by PowerShell, not by the cmdlet
-developer, and they are automatically available to any cmdlet.
+cmdlet. They're implemented by PowerShell, not by the cmdlet developer, and
+they're automatically available to any cmdlet.
 
 You can use the common parameters with any cmdlet, but they might not have an
-effect on all cmdlets. For example, if a cmdlet does not generate any verbose
-output, using the Verbose common parameter has no effect.
+effect on all cmdlets. For example, if a cmdlet doesn't generate any verbose
+output, using the `-Verbose` common parameter has no effect.
 
 The common parameters are also available on advanced functions that use the
-CmdletBinding attribute or the Parameter attribute, and on all workflows.
+`CmdletBinding` attribute or the `Parameter` attribute, and on all workflows.
```

Comment:
Attributes and parameters should be bold not fenced.

> **CmdletBinding** attribute or the **Parameter**

Also, should we be mentioning workflows in v6?

Reply:
Fixed.